app/[locale]/login/__tests__/page.spec.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import {fireEvent, renderWithAuth, screen, waitFor} from '@/utils/test-utils'; //wrapping test with AuthProvider
+import { fireEvent, renderWithProviders, screen, waitFor } from '@/utils/test-utils'; //wrapping test with AuthProvider
```

Comment:
Updated the name of the custom test util to `renderWithProviders` since we added the NextIntlClientProvider